src/atoms/Header/Header.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Refer this comment : https://github.com/Sg1211/todo-app-react/pull/1/files#r906078827